src/mbpfan.c

```diff
@@ -148,71 +152,72 @@ t_sensors *retrieve_sensors()
             }
         }
 
-        path_begin = strdup("/sys/devices/platform/coretemp.0/hwmon/hwmon");
+	// loop over up to 6 processors
+	int processor;
+	for (processor = 0; processor < 6; processor++) {
```

Comment:
Could you extract constants like this into `NUM_PROCESORS`, `NUM_HWMONS`, and `NUM_TEMP_INPUTS`?